azavea/raster-foundry
# Add mosaic/color definition persistence

## Overview

This commit adds endpoints and updates database
models to enable persisting color and mosaic definition
persistence.

The following endpoints were added:
 - /projects/{uuid}/mosaic/ -- lists a projects mosaic definitions
 - /projects/{uuid}/mosaic/{uuid} -- a particular scene's definition
   for a project
 - /projects/{uuid}/order/ -- ordered list of scene ids for a project

Supersedes #804 (I could not update it because it was on a fork)

### Checklist

~- [ ] Styleguide updated, if necessary~

## Testing Instructions

 * How to test this PR
 * Prefer bulleted description
 * Start after checking out this branch
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case, and expected output

Closes #700
Closes #701
Closes #702
